documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -6,6 +6,9 @@ Release Notes
 ======
 * The multi-version client API would not propagate errors that occurred when creating databases on external clients. This could result in a invalid memory accesses. `(PR #5221) <https://github.com/apple/foundationdb/pull/5221>`_
 * Fixed a race between the multi-version client connecting to a cluster and destroying the database that could cause an assertion failure. `(PR #5221) <https://github.com/apple/foundationdb/pull/5221>`_
+* Added Mako latency measurements. `(PR #5255) <https://github.com/apple/foundationdb/pull/5255>`_
+* Fixed a bug when passing InconsistentSnapshotOnly, where it was incorrectly passed to `lockDB` field. `(PR #5228) <https://github.com/apple/foundationdb/pull/5228>`_
```

Comment:
I haven't looked carefully at the PR, but it's still a bit unclear to me what the impact of this bug is. What does the behavior look like for a user of the software? One goal of these release notes is for people to be able to recognize when a problem they've been having has been fixed, and typically that is most easily recognized if the release note is describe in terms of the problem they are having rather than what was actually wrong in the code.

```suggestion
* Fixed a bug when passing InconsistentSnapshotOnly, where it was incorrectly passed to `lockDB` field. `(PR #5228) <https://github.com/apple/foundationdb/pull/5228>`_
```

Reply:
Done. PTAL.